### DISCUSSION TITLE
Tabbed fieldsets and inlines

### DISCUSSION THREAD

**Discussion by elena-yudina** [fabiocaccamo/django-admin-interface#271] [Q&A] [answered]
Hello! How can I add tabbed inlines?

**Comment by fabiocaccamo** [comment, marked as answer]
Hi @elena-yudina 
you just need to activate the option in the theme admin:

- Go to the themes admin: `/admin/admin_interface/theme/`
- Go inside the theme you are using
- Scroll to the bottom, select the options you need, then save

<img width="240" alt="Screenshot 2023-04-13 at 11 25 15" src="https://user-images.githubusercontent.com/1035294/231715962-354fdbc3-6722-4276-a919-e449f9d37f37.png">

**Reply by elena-yudina** [reply]
Thank you!

**Comment by elena-yudina** [comment]
Ok, I've added tabs for inlines, bit it doesn't work :(
I think the reason is that I'm using Russian verbose_name and verbose_name_plural. Is there any way to fix it?

**Reply by fabiocaccamo** [reply]
In this case your issue is related to #237

**Reply by fabiocaccamo** [reply]
@elena-yudina the issue has been fixed in `0.25.0` version, let me know!

**Reply by elena-yudina** [reply]
Now it works perfect! Thank you!
